scripts/flutter-dev-setup.ps1

```diff
@@ -0,0 +1,169 @@
+#requires -version 2
+<#
+.SYNOPSIS
+  This script assists in installing the required build-tools and platform SDKs to get started
+  with Flutter development on an Android Phone.
+.DESCRIPTION
+  This script assists in installing the required build-tools and platform SDKs to get started
+  with Flutter development on an Android Phone. It detects whether any SDKs are installed and
+  prompts to install them if not.
+.INPUTS
+  None
+.OUTPUTS
+  None
+.NOTES
+  Version:        1.0
+  Author:         Ardelean Călin
+  Creation Date:  14 July 2018
+  Purpose/Change: Initial script development
+#>
+
+
+# Gets the installed packages list from sdkmanager
+function get-installed {
```

Comment:
This script could be rewritten to follow core format. But as it is work it is OK for moving.